### ISSUE TITLE
Broken perspective camera in Fusion >= 2.0.3470

### ISSUE THREAD

**Issue by xoniq** [pravdomil/Native-Trackpad#1]
On my MacBook Pro every 2-finger action and every Magic Mouse touch action does zooming in on the model, and only zooming in. I cannot move around, or zoom out anymore.

**Comment by wjatevrr** [comment]
I'm having the same problem. It seems there is a compatibility issue with this plugin and the latest version of Fusion 360. As of last night everything was running fine, then after today's update the plugin appears to be broken. The camera will zoom in whenever paning with the 2 fingers gesture, then it will get stuck and won't allow to zoom out.

**Comment by pravdomil** [comment]
valid, **hot fix:** use orthographic camera

**Comment by pravdomil** [comment]
this seems to be Fusion API issue
here is steps to reproduce, can anybody send it to Autodesk?

``` cpp
// this function produces different results when camera is set to perspective or orthographic
// but in both cases it should do nothing
void panViewportCameraByVector() {
    auto vector = Vector3D::create();
    
    auto camera = app->activeViewport()->camera();
    camera->isSmoothTransition(false);
    
    auto eye = camera->eye();
    eye->translateBy(vector);
    camera->eye(eye);
    
    auto target = camera->target();
    target->translateBy(vector);
    camera->target(target);
    
    app->activeViewport()->camera(camera);
    app->activeViewport()->refresh();
}
```

BTW: what was your experience with this plug-in so far?

**Comment by pravdomil** [comment]
fixed in Fusion 3.0.3700
